src/sentry/api/serializers/models/organization.py

```diff
@@ -180,19 +213,37 @@ def serialize(self, obj, attrs, user):
         }
 
 
-class OnboardingTasksSerializer(Serializer):
-    def get_attrs(self, item_list, user, **kwargs):
+class _OnboardingTasksAttrs(TypedDict):
+    user: Optional[Union[UserSerializerResponse, UserSerializerResponseSelf]]
+
+
+class OnboardingTasksSerializerResponse(TypedDict):
+
+    task: str  # TODO: literal/enum
+    status: str  # TODO: literal/enum
+    user: Optional[Union[UserSerializerResponse, UserSerializerResponseSelf]]
+    completionSeen: str
+    dateCompleted: str
+    data: Any  # JSON object
+
+
+class OnboardingTasksSerializer(Serializer):  # type: ignore
+    def get_attrs(
+        self, item_list: OrganizationOnboardingTask, user: User, **kwargs: Any
+    ) -> MutableMapping[OrganizationOnboardingTask, _OnboardingTasksAttrs]:
```

Comment:
I see what you mean about these being verbose. You can revert this if you want, up to you.

Reply:
no worries, will go ahead and keep them. and added a couple of serializer tests for these models